packages/runtime/src/runtimes/react/controls.tsx

```diff
@@ -47,105 +27,20 @@ import {
-  getWidthPropControllerDataResponsiveLengthData,
-  WidthPropControllerFormat,
-  WidthDescriptor,
-  GapX,
-  ResponsiveNumber,
-  ResponsiveIconRadioGroup,
-  ResponsiveSelect,
-  ResponsiveOpacity,
-  getSocialLinksPropControllerDataSocialLinksData,
-} from '@makeswift/prop-controllers'
-
-import { isLegacyDescriptor } from '../../prop-controllers/descriptors'
-import { useResponsiveLengthPropControllerData } from '../../components/hooks/useResponsiveLengthPropControllerData'
-import { useNumberPropControllerData } from '../../components/hooks/useNumberPropControllerData'
-import { useResponsiveColorPropControllerData } from '../../components/hooks/useResponsiveColorPropControllerData'
-import { useTextStylePropControllerData } from '../../components/hooks/useTextStylePropControllerData'
-import { useNavigationLinksPropControllerData } from '../../components/hooks/useNavigationLinksPropControllerData'
-import { useTextAreaPropControllerData } from '../../components/hooks/useTextAreaPropControllerData'
-import { usePropValue } from '../../components/hooks/usePropValue'
-import { useGapYPropControllerData } from '../../components/hooks/useGapYPropControllerData'
-import { useElementIDPropControllerData } from '../../components/hooks/useElementIDPropControllerData'
-import { useTableFormFieldsPropControllerData } from '../../components/hooks/useTableFormFieldsPropControllerData'
-import { useGridPropControllerData } from '../../components/hooks/useGridPropControllerData'
-import { useImagePropControllerData } from '../../components/hooks/useImagePropControllerData'
-import { useImagesPropControllerData } from '../../components/hooks/useImagesPropControllerData'
-import { useBackgroundsPropControllerData } from '../../components/hooks/useBackgroundsPropControllerData'
-import { useTextInputPropControllerData } from '../../components/hooks/useTextInputPropControllerData'
-
-export type ResponsiveColor = ResponsiveValue<ColorValue>
-
-function useWidthStyle(
-  data: WidthPropControllerData | undefined,
-  descriptor: WidthDescriptor,
-): string {
-  const value = getWidthPropControllerDataResponsiveLengthData(data)
-
-  return useStyle(useResponsiveWidth(value, descriptor.options.defaultValue))
-}
-
-function usePaddingStyle(data: PaddingPropControllerData | undefined): string {
-  const value = getPaddingPropControllerDataResponsivePaddingData(data)
-
-  return useStyle(useResponsivePadding(value))
-}
-
-function useMarginStyle(data: MarginPropControllerData | undefined): string {
-  const value = getMarginPropControllerDataResponsiveMarginData(data)
-
-  return useStyle(useResponsiveMargin(value))
-}
-
-export function useBorderRadiusStyle(data: BorderRadiusPropControllerData | undefined): string {
-  const value = getBorderRadiusPropControllerDataResponsiveBorderRadiusData(data)
-
-  return useStyle(useResponsiveBorderRadius(value))
-}
-
-export function useShadowsStyle(data: ShadowsPropControllerData | undefined): string {
```

Comment:
Looks like these functions are exported previously, but not in the change. Were these exports publicly exposed/consumed outside of this package?

Reply:
Good question, forgot to comment on this, they weren't publicly exposed or used outside of the `controls.tsx` file.